docs/source/reference/kubernetes/multi-kubernetes.rst

```diff
@@ -0,0 +1,146 @@
+.. _multi-kubernetes:
+
+Multi-Kubernetes Clusters
+=============================
+
+
+SkyPilot allows you to manage dev pods, jobs and services across multiple Kubernetes clusters in a single pane of glass.
+
+You may have multiple Kubernetes clusters for different:
+
+* **Use cases:** e.g., a production cluster and a development/testing cluster.
+* **Regions or clouds:** e.g., US and EU regions; or AWS and Lambda clouds.
+* **Accelerators:** e.g., NVIDIA H100 cluster and a Google TPU cluster.
+* **Configurations:** e.g., a small cluster for a single node and a large cluster for multiple nodes.
+* **Kubernetes versions:** e.g., to upgrade a cluster from Kubernetes 1.20 to 1.21, you may create a new Kubernetes cluster to avoid downtime or unexpected errors.
+
+
+.. image:: /images/multi-kubernetes.svg
+    :width: 80%
+    :align: center
+
+.. original image: https://docs.google.com/presentation/d/1_NzqS_ccihsQKfbOTewPaH8D496zaHMuh-fvPsPf9y0/edit#slide=id.p
+
+Configuration
+-------------
+
+Step 1: Set Up Credentials
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+To work with multiple Kubernetes clusters, their credentials must be set up as individual `contexts <https://kubernetes.io/docs/tasks/access-application-cluster/configure-access-multiple-clusters/>`_ in your local ``~/.kube/config`` file. 
+
+For deploying new clusters and getting  credentials, see :ref:`kubernetes-setup-deploy`.
```

Comment:
The reference renders as `Step 1 - Deploy a Kubernetes Cluster`, which gets confusing since this is supposed to be step 1. Maybe change the reference directly to the deployment page.

```suggestion
For deploying new clusters and getting  credentials, see :ref:`kubernetes-deployment`.
```